README.adoc

```diff
@@ -8,6 +8,19 @@ these instructions assume minikube for now, for example:
 minikube start --cpus=4 --memory=4096 --disk-size=64g
 ```
 
+== Build the kubernetes-model jar Enhanced with Sk8s Types
```

Comment:
Seems a bit much to require this step for the demo, seems to require some serious go chops to get his to even work ...

Reply:
See updated instructions, need to be cloned inside $GOPATH